# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../wbar/wbar-1.3.3-gcc43-getenv.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

diff -ruN wbar-1.3.3-orig/IconLoader.cc wbar-1.3.3/IconLoader.cc
--- wbar-1.3.3-orig/IconLoader.cc	2006-12-29 02:54:29.000000000 +0100
+++ wbar-1.3.3/IconLoader.cc	2010-08-26 12:07:09.576054622 +0200
@@ -4,6 +4,8 @@
 #include <sys/types.h>
 };
 
+#include <cstdlib>
+
 #include "IconLoader.h"
 #include "debug.h"
 
diff -ruN wbar-1.3.3-orig/SuperBar.cc wbar-1.3.3/SuperBar.cc
--- wbar-1.3.3-orig/SuperBar.cc	2007-08-04 02:24:34.000000000 +0200
+++ wbar-1.3.3/SuperBar.cc	2010-08-26 12:05:02.844054771 +0200
@@ -1,3 +1,5 @@
+#include <cstdlib>
+
 #include "SuperBar.h"
 #include "SuperIcon.h"
 
